JoaoVictorLouro/resourceful-tako
# Add base stack endpoints

# What does this PR do or change?

- Adds the base stack add and stack deploy API endpoints
- Adds a new cwd field to the stack model

# How to test it, expected behaviour:

- Call the api, it should be able to create new stacks and deploy stacks that do not use any Dockerfiles or relative path volumes

# What could go wrong?

- It does not support any stack with relative paths in it

# New dependencies:

- `yaml`, to validate the stack code

# PR Checklist:

- [x] Added description to the PR
- [x] Added how to test the PR
- [x] Added tests to the new functionality or component
- [x] Linked issue to PR
- [x] Took screenshots
- [x] Analyzed the impact of possible issues that my have arisen from this PR.

# Screenshots of changes:

- N/A
